src/www/vpn_openvpn_export.php

```diff
@@ -551,8 +551,8 @@
                 <td >
                       <input name="openvpnmanager" id="openvpnmanager" type="checkbox" value="yes" />
                       <div class="hidden" for="help_for_openvpnmanager">
-                        <?=gettext('This will change the generated .ovpn configuration to allow for usage of the management interface.'.
-                        'And include the OpenVPNManager program in the "Windows Installers". With this OpenVPN can be used also by non-administrator users.'.
+                        <?=gettext('This will change the generated .ovpn configuration to allow for usage of the management interface. '.
+                        'And include the OpenVPNManager program in the "Windows Installers". With this OpenVPN can be used also by non-administrator users. '.
```

Comment:
@AdSchellevis maybe there are additional changes required as you removed the installers.